graylog2-web-interface/src/routing/Routes.jsx

```diff
@@ -93,6 +93,8 @@ const Routes = {
       USERS: {
         CREATE: '/system/authentication/users/new',
         edit: username => `/system/authentication/users/edit/${username}`,
+        edit_tokens: username => `/system/authentication/users/tokens/${username}`,
```

Comment:
I think it makes sense to group them as a `TOKENS` subkey, as tokens are a subresource of a user.